jupyterbook/content/code_gallery/gallery.md

```diff
@@ -350,6 +350,21 @@
 <img src="../images/2024-05-16-white_shark_water_temp.png"/>
 :::
 
+:::{grid-item-card} Using AphiaIDs to download OBIS occurrences for Bio/Eco EOVs
+:margin: 3 0 0 0
+:link: ./data_analysis_and_visualization_notebooks/2024-09-13-OBIS_EOVs.html
+:link-type: url
+:text-align: center
+:shadow: md
+
+{bdg-primary}`OBIS`
+{bdg-secondary}`EOVs`
+{bdg-info}`R`
```

Comment:
@laurabrenskelle and @MathewBiddle these make the gallery badges. Are these words OK? I'm pretty sure we can come up with better ones but my head shutting down for the night and I cannot come up with better ones now.